### PR TITLE
QuestDB: add types, seek to extend code to find issues in QuestDB

### PR DESCRIPTION
**Pull request by marregui**
I have started playing with the QuestDB kit inside of SQLancer.
At present I am adding tests and I am trying to get my had around of the messages I get when I execute:

```bash
Main questdb
```

My goal is to improve this draft until I feel confident enough to make it into an official PR.

Any early feedback is appreciated.

Note: I will reformat the code at the end. At present I use intellij and it is configured as per QuestDB standards. I will make the code affected by this PR have the same style as the rest of the codebase.

### CODE REVIEW

File: src/sqlancer/mariadb/gen/MariaDBSetGenerator.java [hunk 1]
```diff
@@ -35,11 +35,11 @@ private enum Action {
 
         AUTOCOMMIT("autocommit", (r) -> 1, Scope.GLOBAL, Scope.SESSION), //
         BIG_TABLES("big_tables", (r) -> Randomly.fromOptions("OFF", "ON"), Scope.GLOBAL, Scope.SESSION), //
-        COMPLETION_TYPE("completion_type", (r) -> Randomly.fromOptions("'NO_CHAIN'", "'CHAIN'", "'RELEASE'", 0, 1, 2),
+        COMPLETION_TYPE("completion_type", (r) -> Randomly.fromOptions("'NO_CHAIN'", "'CHAIN'", "'RELEASE'", "0", "1", "2"),
```
Comment (mrigger) [inline src/sqlancer/mariadb/gen/MariaDBSetGenerator.java:38]: What's the motivation for these changes? Any IDE warning or so?

Comment (marregui) [reply src/sqlancer/mariadb/gen/MariaDBSetGenerator.java:38]: hi! yes, warnings in intelliJ. I will resume with the PR soon, another priority kicked in :), you know how these things are.
